### PR TITLE
feat(col): expose browser table methods

### PR DESCRIPTION
**Pull request by david-allison**
https://github.com/ankitects/anki/blob/33a923797afc9655c3b4f79847e1705a1f998d03/pylib/anki/collection.py

https://github.com/ankitects/anki/blob/33a923797afc9655c3b4f79847e1705a1f998d03/pylib/anki/browser.py

## Fixes
* Prep for #11889

## How Has This Been Tested?
Unit tests & non-functional change

## Checklist
- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

**Comment by BrayanDSO** [comment]
Lint has failed

**Review by lukstbit — APPROVED** [review]
LGTM, thanks for the links!

File: AnkiDroid/src/main/java/com/ichi2/libanki/BrowserConfig.kt [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        if (isNotesMode) NOTES_SORT_COLUMN_KEY else CARDS_SORT_COLUMN_KEY
+
+    @LibAnkiAlias("sort_backwards_key")
+    fun sortBackwardsKey(isNotesMode: Boolean): String =
+        if (isNotesMode) NOTES_SORT_BACKWARDS_KEY else CARDS_SORT_BACKWARDS_KEY
+}
+
+object BrowserDefaults {
+    @LibAnkiAlias("CARD_COLUMNS")
+    val CARD_COLUMNS: List<String> = listOf("noteFld", "template", "cardDue", "deck")
```
Comment (david-allison) [inline AnkiDroid/src/main/java/com/ichi2/libanki/BrowserConfig.kt:50]: Different in AnkiMobile, I'll leave this change in for now, and at a later date we can discuss whether we use these in Tablet mode, or stick to the AnkiMobile selection

**Review by mikehardy — APPROVED** [review]
LGTM let's go

**Pull request merged**